flaml/model.py

```diff
@@ -580,6 +588,10 @@ def on_epoch_end(self, args, state, control, **callback_kwargs):
             setattr(self._trainer, "_is_seq2seq", True)
         if kwargs.get("gpu_per_trial"):
             self._trainer.args._n_gpu = kwargs.get("gpu_per_trial")
+
+        if kwargs.get("gpu_per_trial") == 0:
+            os.environ["CUDA_VISIBLE_DEVICES"] = ""
+

```

Comment:
with `no_cuda=True`, is this still needed?

Reply:
yes